py/server/deephaven/dtypes.py

```diff
@@ -228,21 +228,26 @@ def null_remap(dtype: DType) -> Callable[[Any], Any]:
 def _instant_array(data: Sequence) -> jpy.JType:
     """Converts a sequence of either datetime64[ns], datetime.datetime, pandas.Timestamp, datetime strings,
     or integers in nanoseconds, to a Java array of Instant values. """
+
     # try to convert to numpy array of datetime64 if not already, so that we can call translateArrayLongToInstant on
     # it to reduce the number of round trips to the JVM
     if not isinstance(data, np.ndarray):
         try:
-            data = np.array([pd.Timestamp(dt).to_numpy() for dt in data], dtype=np.datetime64)
+            # Pandas drops unrecognized time zones, so it may handle time zones incorrectly
```

Comment:
This comment seems more focused on the current code review than on future maintainability.
```suggestion
            # Pandas drops unrecognized time zones, so it may handle time zones incorrectly when parsing strings
```